app/controllers/attendees_controller.rb

```diff
@@ -2,17 +2,17 @@ class AttendeesController < ApplicationController
   before_action :set_session
 
   def create
-    @session.attendees.push(current_user)
-    flash[:notice] = I18n.t("controllers.attendees.add_user.notice")
-
-    redirect_back_or_to(sessions_path, params: params[:starts_at])
+    @session.attendees.push(current_user) if @session.attendees.exclude?(current_user)
+    @user_session_ids = current_user.sessions.pluck(:id)
   end
 
   def destroy
-    @session.attendees.delete(current_user)
-    flash[:notice] = I18n.t("controllers.attendees.remove_user.notice")
-
-    redirect_back_or_to(sessions_path, params: params[:starts_at])
+    @session.attendees.delete(current_user) if @session.attendees.include?(current_user)
+    @user_session_ids = current_user.sessions.pluck(:id)
+    if URI(request.referer).path == schedule_path
+      flash[:notice] = I18n.t("controllers.attendees.remove_user.notice")
+      redirect_back_or_to(sessions_path, params: params[:starts_at])
```

Comment:
we redirect on My Schedule only because this happens if you remove a session, right?
<img width="972" alt="image" src="https://github.com/user-attachments/assets/c79c951a-5c6d-436a-be5d-fe622cf17e59">